apps/web-giddh/src/assets/magic-link-assets/magic.css

```diff
@@ -222,10 +222,6 @@ body {
     padding-right: 5px;
 }
 
-.pr-0 {
```

Comment:
please check if the class pr-0 is used in magic.html then this css should not be removed as magic.html is not taking bootstrap classes by default

Reply:
@aditi-tiwari17 the styling is there only the position is changed